test/xdr/accounts/thresholds_test.exs

```diff
@@ -0,0 +1,72 @@
+defmodule Stellar.XDR.ThresholdsTest do
+  use ExUnit.Case
+
+  alias Stellar.XDR.{
+    Thresholds,
+    ThresholdsError
+  }
+
+  describe "Thresholds" do
+    setup do
+      master_weight = 128
+      low = 16
+      med = 32
+      high = 64
+      thresholds = Thresholds.new(master_weight: master_weight, low: low, med: med, high: high)
+
+      %{
+        master_weight: master_weight,
+        low: low,
+        med: med,
+        high: high,
+        thresholds: thresholds,
+        binary: <<master_weight, low, med, high>>
+      }
+    end
+
+    test "new/1", %{master_weight: master_weight, low: low, med: med, high: high} do
+      %Thresholds{master_weight: ^master_weight, low: ^low, med: ^med, high: ^high} =
+        Thresholds.new(master_weight: master_weight, low: low, med: med, high: high)
+    end
```

Comment:
IMO, these tests don't need to receive parameters. What about something more direct like:

```elixir
test "new/1" do
  %Thresholds{master_weight: 128, low: 16, med: 32, high: 64} =
    Thresholds.new(master_weight: 128, low: 16, med: 32, high: 64)
end
```

Reply:
Well, having so much named variables just to pass integers around seems noisy. I will apply this change.

Reply:
The tests were refactored to use constants instead of variables.